src/Data/List/Relation/Binary/Permutation/Propositional/Properties.agda

```diff
@@ -33,11 +34,11 @@ open import Relation.Binary.PropositionalEquality.Core as ≡
 open import Relation.Binary.PropositionalEquality.Properties using (module ≡-Reasoning)
 open import Relation.Nullary
 
-private
-  variable
-    a b p : Level
-    A : Set a
-    B : Set b
+private variable
```

Comment:
May not be in `style-guide`, but the additional blankline+indent  is customary everywhere else in `stdlib`...